src/styles/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
```

Comment:
Combine with shorthand `background` property

Reply:
Still not fixed from the previous review